src/server/plugin/AuthCore.ts

```diff
@@ -44,6 +47,6 @@ export class AuthCore {
   }
 
   private getDeniedMessage(username: string) {
-    return `Access denied: User "${username}" is not a member of "${this.requiredOrgName}"`
+    return `Access denied: User "${username}" is not a member of "${this.requiredOrgName}"${this.requiredTeamName ? " or member of " + this.requiredTeamName + " team" : ""}`
```

Comment:
Org names are public, but team names can be private. Since this message is publicly visible, can we change this to an anonymous format? E.g. `${this.requiredTeamName ? " or member of the required team" : ""}`